container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -353,6 +403,49 @@ variables:
   options:
     description: Expiration for generated certificates (in days)
     default: 10950
+- name: DB_EXTERNAL_DRIVER
+  options:
+    immutable: true
+    description: >
+      Database driver to use for the external database server used to manage the
+      CF-internal databases.  Only used if DB_EXTERNAL_HOST is set.  Currently
```

Comment:
```suggestion
      UAA-internal databases.  Only used if DB_EXTERNAL_HOST is set.  Currently
```

Reply:
Will fix.

Reply:
Done with commits 7d897b5a65dc4e5af7d21cf30d83e583a020ed21 and SUSE/scf@addaf660ee8dc5dd16d6def77b307d731dcdbd31